### DISCUSSION TITLE
Windows 11 Steam issue

### DISCUSSION THREAD

**Discussion by mgcarnevali** [kangyu-california/PersistentWindows#306] [Q&A]
Hello,

I bought 2 new monitors and I'm dealing with the display port deep sleep issue on Windows 11. 

First question, Do I have to disable Windows option to remember windows position in order to use PersistentWindows?

In the other hand, my monitors goes to sleep after 10 minutes without use, I move the mouse to wake them but Steam window appears with a very high zoom level as you can see in the attached screenshot, is there any way to fix it?

![steam zoom](https://github.com/kangyu-california/PersistentWindows/assets/38984659/cf514e93-cb7d-4e77-9fb3-ba7d572dab0d)

Unfortunatelly its doesn't work on my computer, all windows are moved to main screen after waking up monitors

**Comment by kangyu-california** [comment]
PW should work fine with Win11 built-in auto restore.
The second issue sounds similar to a closed issue #289, please try latest 5.53 release.

**Comment by mgcarnevali** [comment]
I'm using version 5.53 since this morning, same issue. Not sure why but it does not save the position or the status (minimized/maximized) of any window.

Tray icon blinks red or something similar after waking up the monitors but nothing happen.

Do I need to execute persistenwindows with this command option -dpi_sensitive_call=0?

**Comment by kangyu-california** [comment]
Perhaps Win11 built-in auto restore overrides PW correction. You may try turn off that Win11 feature.

Another workaround is add PW command option -delay_auto_restore 2.5  (adjust the value in seconds here as needed) to force PW to yield for hardware/OS action before starting auto restore.

 -dpi_sensitive_call=0 is phased out, there is no need for that option anymore

**Comment by mgcarnevali** [comment]
Same thing with built in feature turned off, tried this morning after upgrading to latest version.

Also tried -delay_auto_restore 2.5 with no luck, going to exclude steam to check if that helps.

I'm not using it with the scheduled task, just the shortcut in the startup folder but not sure if that makes any difference.

**Comment by mgcarnevali** [comment]
Hello,

I think it is working now, going to test it during the day and confirm,

I made 2 changes, "reinstalled" persistentwindows after deleting its folder from appdata and created the scheduled task, not sure what changed but successfully restored my windows 2 times.

**Comment by mgcarnevali** [comment]
Hello,

After 2 weeks of testing I can confirm that the issue still happens sometimes with Steam.
